meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/ZkMetaClient.java

```diff
@@ -449,12 +449,62 @@ private void cleanUpAndClose(boolean cancel, boolean close) {
     }
   }
 
+  /**
+   * MetaClient uses ZkClient to connect underlying metadata services. ZkClient current do auto
```

Comment:
suggestion: MetaClient uses (which) ZkClient to connect to ZK. Current implementation of ZkClient auto-reconnects infinitely.

Reply:
TFTR. Updated.